src/client/swagger/unity.yml

```diff
@@ -5,14 +5,14 @@ info:
 servers:
   - url: /unity/v2 # to be determined
 paths:
-  '/billing/account':
+  '/account':
     get:
       operationId: GetAccount
       tags:
         - Account
       responses:
         '200':
-          description: Account info for billing
+          description: the account
```

Comment:
Maybe I'm thinking of this too much as a traditional REST API, but it feels really strange everything has this implicit coupling to the user's session. Though I suppose the other option is `/account/:id` and then you're implicitly coupled to the session by way or authorization (checking the logged in user's account id matches the requested one) anyway.

Reply:
Don't 🔫 me but if I were starting from scratch I'd probably lean more towards a use-case based model than a resource based one. Then split out resources if/when duplication becomes apparent. That's personal preference though, and I think the scope of these pages is small enough (for now) that either way will work fine.